### PR TITLE
Hadoop c

### PR DESCRIPTION
**Pull request by ng-hiep**
# add entrypoint.sh and Makefile
@ntphiep #1

### CODE REVIEW

**Review by ng-hiep — COMMENTED** [review]
co cai l gi dau?

**Comment by ng-hiep** [comment]
?

**Review by ntphiep — COMMENTED** [review]
co cl

File: crawl_learn_1/test.py
Comment (ntphiep) [inline crawl_learn_1/test.py:1]: ?????

Comment (ntphiep) [reply crawl_learn_1/test.py:1]: cc

**Review by ntphiep — CHANGES_REQUESTED** [review]

**Comment by ntphiep** [comment]
clm

**Review by ntphiep — COMMENTED** [review]
ok

**Review by ntphiep — COMMENTED** [review]
change nothing

**Review by ntphiep — APPROVED** [review]

**Comment by ng-hiep** [comment]
close

**Review by ng-hiep — COMMENTED** [review]
f

**Pull request closed without merging**